### PR TITLE
Fix Text Editing Issues in VS Code Extension

### PR DESCRIPTION
**Pull request by seflless**
**THIS PR IS NOT READY FOR MERGING**

This addresses https://github.com/tldraw/tldraw/issues/239 and related issues.

**To Fix** (For Stickies and Text components)
 - [x] You can't cut/copy/paste when when text-editing/focused
 - [x] You can't undo/redo when when text-editing/focused 
 - [x] You can't use Command+A to select all the text, when when text-editing/focused

**Smoke Tests**
 - Desktop web app text editing
 - Mobile web app text editing
 - Multiplayer text editing
 - VS Code desktop, vscode.dev, github.dev, and Codespaces
 - Component demo?

### CODE REVIEW

File: packages/tldraw/src/state/shapes/StickyUtil/StickyUtil.tsx [hunk 1]
```diff
@@ -126,6 +127,7 @@ export class StickyUtil extends TDShapeUtil<T, E> {
 
       // Resize to fit text
       React.useEffect(() => {
+        console.log('hi!');
```
Comment (cagataycali) [inline packages/tldraw/src/state/shapes/StickyUtil/StickyUtil.tsx:130]: Please remove this console.log, it looks like you forget this here

Comment (seflless) [reply packages/tldraw/src/state/shapes/StickyUtil/StickyUtil.tsx:130]: Hey, this PR isn't ready for merging. Sorry for the confusion. I add **THIS PR IS NOT READY FOR MERGING** to the PR description. That's in there for debugging reasons at the moment, and I'll remove when I get closer to being ready for a final review.

**Comment by seflless** [comment]
I'm going to leave notes/learnings/questions about this PR here

**Comment by seflless** [comment]
I suspect the Tldraw shortcut keys system should just be letting events that Textareas handle untouched. 

The right click menu should probably be the native menu when right clicking on top of text actively being edited.

Us (Regardless of in edit mode or not)
Screen Shot 2021-11-27 at 12 51 04 PM

Excalidraw (when in text edit mode)
Screen Shot 2021-11-27 at 12 47 12 PM

Excalidraw (when not in text edit mode)
Screen Shot 2021-11-27 at 12 47 16 PM

**Comment by seflless** [comment]
I suspect there's going to be accessibility or localization issues with text editing if we don't try to stay as hands off as possible with event handling when the user is editing text.

**Pull request merged**